### PR TITLE
All Entities CRUD APIs created

### PR DESCRIPTION
**Pull request by tushar-kendre**

### CODE REVIEW

**Review by hemant1026 — APPROVED** [review]

File: code/packages/database/prisma/migrations/20241024174745_removed_date_fields_from_recruiter_shortlist/migration.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `createdAt` on the `RecruiterShortlist` table. All the data in the column will be lost.
+  - You are about to drop the column `deletedAt` on the `RecruiterShortlist` table. All the data in the column will be lost.
+  - You are about to drop the column `updatedAt` on the `RecruiterShortlist` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "RecruiterShortlist" DROP COLUMN "createdAt",
```
Comment (faizanurfi) [inline code/packages/database/prisma/migrations/20241024174745_removed_date_fields_from_recruiter_shortlist/migration.sql:10]: Why were these removed? @devilsbane98

**Review by faizanurfi — APPROVED** [review]
